.travis.yml

```diff
@@ -7,11 +7,17 @@ env:
     - CUDA_VERSION="10.0"
     - CUDA_PKG_VERSION="130-1"
     - CUDA_PKG=cuda-repo-ubuntu1404_${CUDA_VERSION}.${CUDA_PKG_VERSION}_amd64.deb
-    - CUDA_HOME=/usr/local/cuda
-    - PATH=${PATH}:${CUDA_HOME}/bin:${GOPATH}/bin
-    - LD_LIBRARY_PATH=${CUDA_HOME}/lib:${TRAVIS_HOME}/gopath/src/github.com/uber/aresdb/lib
+    - CUDA_LIB_LOCAL=${HOME}/cuda_lib
```

Comment:
maybe better to have version in the folder name so that later when we change cuda version, we don't need to manually delete the cache.

Reply:
good point, will update